doc/script_commands.txt

```diff
@@ -9747,6 +9748,7 @@ mapflag%TAB%<map_name>%TAB%battleground%TAB%2
 
 This command will create a new BG Team.
 When player dies, they will be respawned map_name,X,Y as mentioned.
+If the map_name is "-", player don't automatically respawn on the battleground.
```

Comment:
since the script command are verifying if it is a valid map, does it really necessary to force using `-` here? 
empty map name would do? or any invalid map name too.

Reply:
only "-"
```
static BUILDIN(bg_create_team)
{
	const char *map_name, *ev = "", *dev = "";//ev and dev will be dropped.
	int x, y, map_index = 0, bg_id;

	map_name = script_getstr(st,2);
	if( strcmp(map_name,"-") != 0 ) {
		map_index = script->mapindexname2id(st,map_name);
		if( map_index == 0 ) { // Invalid Map
			script_pushint(st,0);
			return true;
		}
	}
```
aleos understand what I'm saying there
https://github.com/rathena/rathena/pull/3967

Reply:
oh i see, overlooked that. Anyway the doc at rA are seem to be easier to understand, or at least the intention of using `"-"` in the parameter.